app/main.py

```diff
@@ -16,4 +16,10 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+mutable = [marks, collection_of_coins, my_favourite_films]
+immutable = [lucky_number, pi, one_is_a_prime_number, name, profile_info]
+
+sorted_variables = {
+    "mutable": mutable,
+    "immutable": immutable
+}
```

Comment:
Your approach is great, but we should adhere examples if possible, so in example we see directly categorized into the sorted_variables dictionary variables, follow the examples, please.